Cargo.toml

```
@@ -87,6 +87,7 @@ thiserror = "1.0"
 time = "0.3"
 tokio = { version = "1.20", features = ["fs"] }
 tracing = { version = "0.1", optional = true }
+suppaftp = {version = "^4.4.0", features = ["secure"]}
```

```suggestion
suppaftp = {version = "4.4", features = ["secure"]}
```

Since we are introducing a new dep here, please consider hide FTP support under a new feature flag `services-ftp`. And maybe we need to enable `async-secure` instead?